### PR TITLE
Kedro notebook

### PR DESCRIPTION
**Pull request by astrojuanlu**

### CODE REVIEW

File: solutions/nb03_ex01.py
Comment (deepyaman) [inline solutions/nb03_ex01.py:1]: Should this be ex02?

Comment (deepyaman) [reply solutions/nb03_ex01.py:1]: Given the first exercise is the catalog loading.

**Review by deepyaman — APPROVED** [review]
Skimmed through, looks good overall, one quick thing I noticed

File: solutions/nb03_ex01.py
Comment (astrojuanlu) [reply solutions/nb03_ex01.py:1]: Oh true, leftover. Changing and merging.

**Pull request merged**